### PR TITLE
Add page to delete years and its data

### PR DESCRIPTION
**Pull request by magonzalez9**

### CODE REVIEW

File: src/screens/ManageYears.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      cancelButtonText: 'No'
+    });
+
+
+    if(!confirmResult.value) return;
+
+    // Delete all student data from school year
+    removeStudentData = store.removeAllStudentsFromYear(schoolYear); 
+    await removeStudentData; 
+    if(removeStudentData.resolved){
```
Comment (ryan-peterson-emberex) [inline src/screens/ManageYears.js]: Why wouldn't it be?

File: src/screens/ManageYears.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  background-color: #D43425;
+  width: 50%;
+  font-family: "Oswald";
+  font-size: 16px;
+  line-height: 21px;
+  font-weight: 200;
+  margin-top: 25px; 
+`;
+
+const StartFormRoot = styled.div`
```
Comment (ryan-peterson-emberex) [inline src/screens/ManageYears.js]: So a `styled(Column)` 😛

File: src/screens/ManageYears.js (hunk 1 above)
Comment (ryan-peterson-emberex) [reply src/screens/ManageYears.js]: Also, what about if there is an error?

File: src/screens/ManageYears.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+  @action.bound async handleDeleteYearSubmit() {
+    let {removeStudentData, removeSchoolYear} = this; 
+    const{ store } = this.props; 
+
+    const schoolYear = await store.fetchSchoolYear(this.year);
+
+    const confirmResult = await swal({
+      title: `Are you sure you want to delete school year ${schoolYear.year} - ${schoolYear.year+1}?`,
+      text: `Deleting this school year will permanently delete all of the student data associated .`,
```
Comment (jf-emberex) [inline src/screens/ManageYears.js]: "... all of the associated student data." 
I think that reads better.

**Pull request merged**